### DISCUSSION TITLE
Accrescent App Store Availability

### DISCUSSION THREAD

**Discussion by NorthernMagic** [spacecowboy/Feeder#204] [Ideas]
Would it be possible for you to publish your RSS reader app on the Accrescent App Store for Android alongside your other distribution channels?

Many users, especially on privacy and security focused Android Oses like GrapheneOS or those simply privacy conscious, use Accrescent as a way to download their apps securely without relying on Google services.

Many of us in the Graphene community for example use RSS readers like Feeder as a way to track news and it would be highly appreciated for you to submit a publishing application to make it available to Accrescent users.

[https://accrescent.app/](https://accrescent.app/)

[https://accrescent.app/docs/guide/getting-started/index.html](https://accrescent.app/docs/guide/getting-started/index.html)

**Comment by spacecowboy** [comment]
I'm not against it, as long as it can be 100% automated.

Why can't you use F-Droid?

**Reply by matchboxbananasynergy** [reply]
Accrescent will eventually have a way for developers to push updates automatically. There's work being done towards that.

**Reply by ghost** [reply]
Where are we at with automation ? I would really like to be able to install Feeder from Accrescent.

**Comment by spacecowboy** [comment]
I read the link you provided and it seems to be a manual process to update the app :(

**Reply by NorthernMagic** [reply]
Ah thank you for the reply. I understand the concerns around automatic updates.

As for why not using F-Droid. There are an increasing amount of people voicing concerns around F-Droid's general security practices. The official client for example, they provide on their website is an outdated version, still targets an old version of Android, and the lack of automated background updates is a major painpoint. 

I know the client issues can be somewhat mitigated by using an alternate client such as Droidify. But most **new** people coming to F-Droid (most likely influenced by a random YouTube video) will use the official client anyway because that is what the source tells them to use.

There are inherent infrastructure concerns around how F-Droid distributes their apps and while I'm not going to list them as this thread is getting quite long as is. Here is a source highlighting these concerns (many of which still valid today, unfortunately). But the bulk of the concerns lie in how F-Droid violate the security model.

[https://privsec.dev/posts/android/f-droid-security-issues/#looking-to-the-future](https://privsec.dev/posts/android/f-droid-security-issues/#looking-to-the-future)

I'm not saying F-Droid is dying or anything, and people should ditch it in droves (I'm all for people using their own methods to get things). But it would be nice for developers to offer their apps through a much more secure store especially as it grows for those more security conscious that don't want to use the Play Store.

Finally I asked because why not ask. While the app store itself is still in early Alpha, and publishing is a bit of a hassle currently. I felt like helping increase adoption among developers early on despite these hassles. Could yield some benefits for Accrescent users as the store currently lacks any type of RSS reader and introducing one early could convince more people to switch to your reader because its now available through a more secure app storefront. And help speed up development on your end as there are potentially more people to help suggest features and file bug reports.

![Screenshot_20240204-071000.png](https://github.com/spacecowboy/Feeder/assets/116324840/e93a17c1-3dee-4eb3-bc38-c1dbb78b0bbe)

![Screenshot_20240204-072739.png](https://github.com/spacecowboy/Feeder/assets/116324840/b13a48ec-84ed-496f-b2b3-6a5dacd4e81c)

_There are only 11 apps available as shown in the screenshots above. Apps showing new functionality would be massively appreciated._

Thank you for reading my discussion post and I hope you have a great rest of your day spacecowboy! 😁

**Reply by shuvashish76** [reply]
https://accrescent.app/faq#open-source
Personally I prefer F-Droid because unlike Accrescent they accept _only FOSS_ apps.
But I wish best for the project, it has good intentions, no harm in publishing multiple alternative app stores.
For those who want fastest updates can use [Obtanium](https://github.com/ImranR98/Obtainium) anyway.

**Comment by Mr-Bajs** [comment]
Version 1.1.19 (current) supports automatic background updates.

**Comment by dotsilva** [comment]
That would be great.
